build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework:spring-jdbc'
 	implementation 'org.hsqldb:hsqldb'
+	implementation 'org.apache.commons:commons-math3:3.6.1'
```

Comment:
[Meta: I'm always hesitant to add dependencies to a project, but Apache Commons is super reasonable, and this seems like a valid use case]